.github/workflows/nightly-update-packages.yml

```diff
@@ -2,6 +2,12 @@ name: Update Packages
 
 on:
   workflow_dispatch:
+    pull_request:
```

Comment:
This formatting might make GHA unhappy. Pull request should be on the same tab break as workflow_dispatch and schedule. should `- reopened` also be in the list?